<ui version="4.0" >
 <class>DataImportWidgetClass</class>
 <widget class="QWidget" name="DataImportWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Data Import</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" >
   <property name="leftMargin" >
    <number>4</number>
   </property>
   <property name="topMargin" >
    <number>4</number>
   </property>
   <property name="rightMargin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Table : </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QComboBox" name="tableCmb" >
       <property name="minimumSize" >
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tableDescLbl" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Source : </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QComboBox" name="from" >
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="file_2" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Field Mapping</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" >
       <property name="margin" >
        <number>4</number>
       </property>
       <property name="spacing" >
        <number>4</number>
       </property>
       <item row="0" column="0" colspan="3" >
        <widget class="QTableWidget" name="fields" >
         <column>
          <property name="text" >
           <string>Target Field Name</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Source Mapping Column</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QPushButton" name="get" >
         <property name="text" >
          <string>Get</string>
         </property>
         <property name="icon" >
          <iconset resource="RoseApple.qrc" >
           <normaloff>:/RoseApple/Resources/play.png</normaloff>:/RoseApple/Resources/play.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer name="progSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="deleteAll" >
         <property name="text" >
          <string>Delete All</string>
         </property>
         <property name="icon" >
          <iconset resource="RoseApple.qrc" >
           <normaloff>:/RoseApple/Resources/delete.png</normaloff>:/RoseApple/Resources/delete.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="logTab" >
       <attribute name="title" >
        <string>Log</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QTextEdit" name="log" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dataTab" >
       <attribute name="title" >
        <string>Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QTableView" name="datas" />
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QProgressBar" name="progressBar" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value" >
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="RoseApple.qrc" >
         <normaloff>:/RoseApple/Resources/delete.png</normaloff>:/RoseApple/Resources/delete.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="RoseApple.qrc" />
 </resources>
 <connections/>
</ui>
